test/server/utils/runOpen.test.js

```diff
@@ -255,5 +255,31 @@ describe('runOpen util', () => {
         `);
       });
     });
+
+    it('on specify URL with page in Google Chrome incognito mode and log error ', () => {
+      return runOpen(
+        'https://example.com',
+        {
+          open: { app: ['Google Chrome', '--incognito'] },
+          openPage: '/index.html',
+        },
+        logMock
+      ).then(() => {
+        expect(logMock.warn.mock.calls[0][0]).toMatchInlineSnapshot(
+          `"Unable to open \\"https://example.com/index.html\\" in browser: \\"{\\"app\\":[\\"Google Chrome\\",\\"--incognito\\"]}\\". If you are running in a headless environment, please do not use the --open flag"`
+        );
```

Comment:
We need better test :smile:

Reply:
I think the best way to test it is manually because even if I found a way to test chrome the chrome start name is different in operating systems ex : 
from https://www.npmjs.com/package/open
```
The app name is platform dependent. Don't hard code it in reusable modules. For example, Chrome is google chrome on macOS, google-chrome on Linux and chrome on Windows.
```
This is a demo from my machine.
![Peek 2020-04-01 18-49](https://user-images.githubusercontent.com/18349358/78164183-a6514500-7449-11ea-9789-bcca762bb0e7.gif)

Reply:
maybe we can use other flag, not good idea keep errors in tests in that case

Reply:
@evilebottnawi what about replacing it with: 
`expect(opn).toBeCalledWith('https://example.com/index.html',{ app: ['Google Chrome', '--incognito'] });`

Reply:
Good idea :+1: